### DISCUSSION TITLE
Expand options for Window Placements

### DISCUSSION THREAD

**Discussion by TwentySevenCode** [f1multiviewer/issue-tracker#334] [Feature requests]
As a user, I want to be able to view all the windows possible and have them align correctly on my screen. The current implementation is limited to a 3x3 grid, which eliminates anyone who wants to view more than that amount of windows at once from cleanly using this feature. With 20 drivers, and assuming the user has a 16:9 screen in either portrait or landscape the assumed screen layout would be either 4x5 or 5x4. To account for both formats that may want a max of 5 videos either vertically in portrait or horizontally in landscape we can assume 5x5 to be the max. 

I have two naming conventions I drew out for menus, one based off the current(ish) verbiage, and another based off the row of the video (this actually cleaned it up much more than I anticipated). They are  in the attached PDF. R1, R2, etc, denotes the row number.
[f1mv layout suggestion.pdf](https://github.com/f1multiviewer/issue-tracker/files/10808639/f1mv.layout.suggestion.pdf)

**Comment by jeppeolesen** [comment]
Marking this as a duplicate of #20
